app-creator/client/src/utils/palettes.ts

```diff
@@ -0,0 +1,42 @@
+import { Palette } from '../redux/types/enums';
+
+type PaletteMap = {
```

Comment:
Is it possible to turn this into a `Record`? See https://www.typescriptlang.org/docs/handbook/utility-types.html#recordkt